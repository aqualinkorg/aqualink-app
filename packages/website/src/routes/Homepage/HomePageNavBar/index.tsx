import React from "react";
import { AppBar, Toolbar, Grid, Button } from "@material-ui/core";

const HomePageNavBar = () => {
  return (
    <AppBar position="static">
      <Toolbar>
        <Grid item xs={12}>
          <Grid container justify="flex-end" alignItems="center">
            <Grid item>
              <Button variant="contained" color="secondary">
                Log In
              </Button>
            </Grid>
          </Grid>
        </Grid>
      </Toolbar>
    </AppBar>
  );
};

export default HomePageNavBar;
