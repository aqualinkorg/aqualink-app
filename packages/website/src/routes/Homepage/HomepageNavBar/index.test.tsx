import React from "react";
import { render } from "@testing-library/react";

import HomePageNavBar from ".";

test("renders as expected", () => {
  const { container } = render(<HomePageNavBar />);
  expect(container).toMatchSnapshot();
});
