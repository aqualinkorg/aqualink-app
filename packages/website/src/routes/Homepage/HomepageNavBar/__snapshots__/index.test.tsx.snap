// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div>
  <header
    class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-flex-end"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                Log In
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </header>
</div>
`;
